test/integration/integration.h

```diff
@@ -116,7 +116,6 @@ class BaseIntegrationTest : Logger::Loggable<Logger::Id::testing> {
 
   void SetUp();
 
-  // TODO(alyssawilk) port the rest of the tests to v2 and make this non-virtual.
```

Comment:
Admittedly this is still virtual, but now subclasses which are doing any real work are required to call it, as validated in lookupPort, so I claim the TODO is done in spirit :-)